src/v/cluster/topics_frontend.h

```diff
@@ -45,6 +45,11 @@ class topics_frontend {
     ss::future<std::vector<topic_result>> create_topics(
       std::vector<topic_configuration>, model::timeout_clock::time_point);
 
+    ss::future<std::vector<topic_result>>
+    create_non_replicable_topics_on_controller_leader(
```

Comment:
nit: We could follow the same naming scheme as create_topics w.r.t. this behavior. `autocreate_non_replicable_topics`